src/reputation/views/withdrawal_view.py

```diff
@@ -214,11 +226,12 @@ def calculate_transaction_fee(self, network="ETHEREUM"):
             )
             json = res.json()
             gas_price = json.get("result", {}).get("SafeGasPrice", 40)
-            gas_limit = 120000.0
+
+        gas_limit = 120000.0
```

Comment:
Nit: Does it make sense to add a comment or use a constant for this magic number?